gradio/events.py

```diff
@@ -643,6 +643,17 @@ def inner(*args, **kwargs):
         event_trigger.event_name = _event_name  # type: ignore
         event_trigger.has_trigger = _has_trigger  # type: ignore
         event_trigger.callback = _callback  # type: ignore
+        event_trigger.connection = _connection  # type: ignore
+        event_specific_args = (
+            [
+                d["name"]
+                for d in _event_specific_args
+                if d.get("component_prop", "true") != "false"
```

Comment:
What does this `component_prop` attribute mean?

Reply:
Whether or not to pass this arguments (`stream_every` or `time_limit`) it to svelte component in the front-end. Only `stream_every` is used in the component code. `time_limit` is used by statustracker.